Dockerfile

```diff
@@ -1,32 +1,24 @@
-FROM node:14.3.0-alpine as builder
-
+FROM node:12.16.1-alpine as builder
+USER root
 WORKDIR /opt/auth-service
-
 RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool autoconf automake \
     && cd $(npm root -g)/npm \
     && npm config set unsafe-perm true \
     && npm install -g node-gyp 
-
-# Copy across package.json and package-lock.json
-COPY package.json package-lock.json* ./
+COPY package.json package-lock.json* /opt/auth-service/
 RUN npm ci
-# check in .dockerignore what is skipped during copy
-COPY . .
-
-# Create empty log file & 
+COPY . /opt/auth-service
+FROM node:12.16.1-alpine
+WORKDIR /opt/auth-service
+# Create empty log file & link stdout to the application log file
 RUN mkdir ./logs && touch ./logs/combined.log
-
-# link stdout to the application log file
 RUN ln -sf /dev/stdout ./logs/combined.log
-
-# USER node 
-# copy bundle
-# COPY --chown=node --from=builder /opt/auth-service/ .
-
+# Create a non-root user: as-user
+RUN adduser -D as-user 
+USER as-user
+COPY --chown=as-user --from=builder /opt/auth-service .
 # cleanup
-RUN apk del build-dependencies
-# RUN npm prune --production
-
-RUN npm run test:unit
+#RUN apk del build-dependencies
+#RUN npm prune --production
 EXPOSE 4004
-CMD ["npm", "run", "start"]
+CMD ["npm", "run", "start"]
```

Comment:
Hmm maybe we need an editorconfig file in this repo or something... there probably should be a newline here